### ISSUE TITLE
Mistake in src/Client/Html/Cms/Page/Cataloglist/Standard.php

### ISSUE THREAD

**Issue by TheStrangeProgrammer** [aimeos/ai-cms-grapesjs#35]
```
if( !$products->isEmpty() && (bool) $config->get( 'client/html/catalog/lists/stock/enable', true ) === true ) {
					$tview->itemsStockUrl = $this->getStockUrl( $tview, $products );
				}
```
should be 
```
if( !$products->isEmpty() && (bool) $config->get( 'client/html/catalog/lists/stock/enable', true ) === true ) {
					$tview->itemsStockUrl = $this->getStockUrl( $tview, $articles );
				}
```
because it makes variations not be able to be bought when you enable basket-add for cataloglist

**Comment by aimeos** [comment]
You are right, can you create a PR to fix the issue?

**Comment by TheStrangeProgrammer** [comment]
On which branch should I do PR's in the future, master or latest stable

**Comment by aimeos** [comment]
Always on master please, we will merge the change to all relevant branches afterwards

**Comment by TheStrangeProgrammer** [comment]
did it

**Comment by aimeos** [comment]
Merged into 2024.07 and 2023.10. New versions will be tagged with the next release
